src/services/VerifiableCredentialService.ts

```diff
@@ -54,56 +54,40 @@ export class VerifiableCredentialService {
 		};
 	}
 
-	getFailureEvidenceBlock(vendorUuid: string, ciRequired?: boolean): VerifiedCredentialEvidence {
-		if (ciRequired) {
-			return {
-				type: Constants.IDENTITY_CHECK,
-				txn: vendorUuid,
-				strengthScore: 3,
-				validityScore: 0,
-				failedCheckDetails: [
-					{
-						checkMethod: "data",
-						identityCheckPolicy: "none",
-					},
-				],
-				ci: process.env.USE_MOCKED ? mockCI : [
-					"D15",
-				],
-			};
-		} else {
-			return {
-				type: Constants.IDENTITY_CHECK,
-				txn: vendorUuid,
-				strengthScore: 3,
-				validityScore: 0,
-				failedCheckDetails: [
-					{
-						checkMethod: "data",
-						identityCheckPolicy: "none",
-					},
-				],
-			};
-		}
+	getFailureEvidenceBlock(vendorUuid: string, cis?: string[]): VerifiedCredentialEvidence {
+		return {
+			type: Constants.IDENTITY_CHECK,
+			txn: vendorUuid,
+			strengthScore: 3,
+			validityScore: 0,
+			failedCheckDetails: [
+				{
+					checkMethod: "data",
+					identityCheckPolicy: "none",
+				},
+			],
+			...(cis ? { ci: process.env.USE_MOCKED ? mockCI : cis } : {}),
+		};
 	}
 	
 
 	async generateSignedVerifiableCredentialJwt(
 		sessionItem: ISessionItem, nameParts: PersonIdentityNamePart[], birthDate: PersonIdentityBirthDate[], bankAccountInfo: BankAccountInfo, getNow: () => number): Promise<{ signedJWT: string; evidenceInfo: VerifiedCredentialEvidence }> {
 		const now = getNow();
 		const subject = sessionItem.subject;
-		let ciRequired = true;
 
+		// To be removed
+		let cis = sessionItem?.cis;
```

Comment:
We can remove this whole block right? And also remove `warningsErrors` from the session interface?

Reply:
Yeah I was going to do that in a second PR I was just being safe in case we had any journeys in flight when this is deployed